### PR TITLE
New Index Page Added

### PR DESCRIPTION
**Pull request by tmo66**
@garg3133 Sir, as you said, I am opening this PR for #39 and all the changes are relevant to #39 only, kindly check this and provide your valuable feedback.

### CODE REVIEW

**Comment by tmo66** [comment]
@garg3133 Check this PR now!

File: home/urls.py [hunk 1]
```diff
@@ -2,8 +2,10 @@
 from . import views
 
 app_name = 'home'
+
 urlpatterns = [
-    path('', views.index, name='index'),
+    path('new/', views.index, name='new_index'),
+    path('', views.old_index, name='index'),
```
Comment (garg3133) [inline home/urls.py:8]: Old page should use `views.index` while new page should use `views.new_index`.

**Comment by garg3133** [comment]
Also, can you make please make a PR for `Initiatives` issue first as your new index page uses `Initiatives` model.

You've also not transferred some files to `new_home` or similar folder. Please also do that.

File: home/urls.py (hunk 1 above)
Comment (tmo66) [reply home/urls.py:8]: Ohh Sorry, that's my bad....I completely forgot that

**Comment by tmo66** [comment]
No, my new index page has nothing related to Initiatives, instead, my new index function has initiatives, I removed that...

File: home/urls.py [hunk 2]
```diff
@@ -2,7 +2,9 @@
 from . import views
 
 app_name = 'home'
+
 urlpatterns = [
+    path('new/', views.new_index, name='new_index'),
```
Comment (garg3133) [inline home/urls.py:7]: Put this route below `index` page route.

**Review by garg3133 — COMMENTED** [review]
After you're done with this, you may squash all the commits into a single commit so as to keep the commit history clean. You may read about squashing from https://zulip.readthedocs.io/en/latest/git/fixing-commits.html and https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History. If you face any problem, you may ping me.

Tip: Before trying any new thing that changes the commit history, create a new branch and try that out in that and after you're confident, do it in your original branch. It will help you not to lose all your work due to any mistake.

**Pull request closed without merging**